app/views/posts/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= stylesheet_link_tag('new-post')%>
+
+<h2>Add new post</h2>
+<%= form_with model: @post, url: user_posts_path(@user) do |f| %>
+  <div class="form-container">
+  <label for="post_title">title:</label>
+    <%= f.text_field :title, required: true, autofocus: true, class: "form-input", placeholder: "Enter post the title" %>
+    <label for="post_title">text:</label>
+    <%= f.text_area :text, required: true, autofocus: true, class: "form-input", placeholder: "Enter post the body"%>
+    <%= f.submit "Create", class: "create-btn" %>
+  </div>
+<% end %>
```

Comment:
- [ ] [OPTIONAL] This form is not displayed in the views kindly consider including it in the user show page so users can click a button and redirect to it to create a new post

Reply:
Thanks a lot for your optional suggestion @Ismailco  I will implement it once i get the code approved.